templates/filebeat/2/rancher-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+catalog:
+  name: "filebeat-alpine"
+  version: "6.1.1"
+  description: "Catalog's rancher-compose"
+  minimun_rancher_version: v1.0.1
+  maximun_rancher_version: v1.6.12
+  upgrade_from: "1.0.0"
+  questions:
+    - variable: "ENV"
+      label: "Environment Name"
+      description: "Environment to distinguish data source"
+      default: "QA"
+      required: true
+      type: "enum"
+      options:
+        - QA
+        - PP
+        - INT
+        - PROD
```

Comment:
It's nice to have predefined values, but what happens if my environments notation is different?? Mey would be better to use an string type to allow free input. What do you think??